torchgeo/datasets/l8biome.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8 Biome dataset."""
+
+import glob
+import os
+import re
+from typing import Any, Callable, Dict, List, Optional, Sequence, cast
+
+import matplotlib.pyplot as plt
+import torch
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import BoundingBox, download_url, extract_archive
+
+
+class L8Biome(RasterDataset):
+    """L8 Biome dataset.
+
+    The `L8 Biome <https://landsat.usgs.gov/landsat-8-cloud-cover-assessment-validation-data>`__ dataset
+    is a cloud validation dataset of Pre-Collection Landsat 8
+    Operational Land Imager (OLI) Thermal Infrared Sensor (TIRS)
+    terrain-corrected (Level-1T) scenes.
+
+    Dataset features:
+
+    * images evenly divided between eight unique biomes
+    * 5 cloud cover categories
+
+    Dataset format:
+
+    * Images are stored as GeoTIFF files corresponding to different bands
+    * Each cloud mask is in ENVI binary format.
+
+    Dataset classes:
+
+    0. Fill
+    1. Cloud Shadow
+    2. Clear
+    3. Thin Cloud
+    4. Cloud
+
+    If you use this dataset in your research, please cite the following:
+
+    * https://doi.org/10.5066/F7251GDH
+    * https://doi.org/10.1016/j.rse.2017.03.026
+
+    .. versionadded:: 0.5
+    """  # noqa: E501
+
+    url = "https://huggingface.co/datasets/torchgeo/l8biome/resolve/main/{}.tar.gz"  # noqa: E501
+
+    filenames_to_md5 = {
+        "barren": "bb446fda3f6af50930849bb135e99f9c",
+        "forest": "21505d878abac830890ea84abddc3c46",
+        "grass_crops": "33d0c553357f5a439aa85a45916ac89a",
+        "shrubland": "f19afc6dfa818ee3868e7040441d4c6d",
+        "snow_ice": "d7b56084e6267ee114419efdc7f664c9",
+        "urban": "b5f6aabbb380e108c408a8ea5dae3835",
+        "water": "d143049ef64e6e681cea380dd84680e9",
+        "wetlands": "bff0d51db84e26a2a8e776c83ab2d331",
+    }
+
+    # https://gisgeography.com/landsat-file-naming-convention/
+    filename_glob = "LC8*_B1.TIF"
+    filename_regex = r"""
+        ^LC8
+        (?P<wrs_path>\d{3})
+        (?P<wrs_row>\d{3})
+        (?P<date>\d{7})
+        (?P<gsi>[A-Z]{3})
+        (?P<version>\d{2})
+        _(?P<band>B\d{1,2})
+        \.TIF$
+    """
+    date_format = "%Y%j"
+
+    separate_files = True
+    rgb_bands = ["B4", "B2", "B3"]
```

Comment:
```suggestion
    rgb_bands = ["B4", "B3", "B2"]
```
https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C02_T1_TOA

Reply:
Can you remake the plot in the description too?

Reply:
Updated plot